aries_cloudagent/revocation/models/issuer_rev_reg_record.py

```diff
@@ -144,7 +148,7 @@ async def generate_registry(self, context: InjectionContext):
         issuer: BaseIssuer = await context.inject(BaseIssuer)
         tails_hopper_dir = indy_client_dir(join("tails", ".hopper"), create=True)
 
-        LOGGER.debug("create revocation registry with size:", self.max_cred_num)
+        LOGGER.debug(f"create revocation registry with size: {self.max_cred_num}")
```

Comment:
I think this change throws warnings in some linters, the logger functions are meant to take the format arguments separately so the string formatting can be skipped if the log level is higher